service-compiler/src/main/java/io/techery/janet/JanetHttpProcessor.java

```diff
@@ -31,28 +34,31 @@
     private Messager messager;
     private ClassValidator classValidator;
     private HttpActionValidators httpActionValidators;
-    private HelpersFactoryGenerator helpersFactoryGenerator;
+    private HttpHelpersFactoryGenerator httpHelpersFactoryGenerator;
     private HttpHelpersGenerator httpHelpersGenerator;
     private Types typesUtil;
+    private Options options;
+
+    private boolean processed;
 
     @Override
     public synchronized void init(ProcessingEnvironment processingEnv) {
         super.init(processingEnv);
+        options = new Options(processingEnv.getOptions());
         elementUtils = processingEnv.getElementUtils();
         messager = processingEnv.getMessager();
         typesUtil = processingEnv.getTypeUtils();
         classValidator = new ClassValidator(HttpAction.class);
         httpActionValidators = new HttpActionValidators();
         Filer filer = processingEnv.getFiler();
-        helpersFactoryGenerator = new HelpersFactoryGenerator(filer);
+
+        httpHelpersFactoryGenerator = new HttpHelpersFactoryGenerator(filer, findOtherHelpersFactories(), options);
         httpHelpersGenerator = new HttpHelpersGenerator(filer);
     }
 
     @Override
     public Set<String> getSupportedAnnotationTypes() {
-        Set<String> annotataions = new HashSet<String>();
-        annotataions.add(HttpAction.class.getCanonicalName());
-        return annotataions;
+        return Collections.singleton("*");
```

Comment:
Because, sometimes we have no `@HttpAction` annotated classes but have to search for other factories. Processor doesn't call `process` if there is no classes annotated with `@SupportedAnnotationTypes` values

I'm sorry didn't mentioned this with pr description.